host/windows/syscall.c

```diff
@@ -324,6 +326,449 @@ __declspec(noreturn) static void _panic(
 **==============================================================================
 */
 
+// Allocates char* string which follows the expected rules for
+// enclaves. Paths in the format
+// <driveletter>:\<item>\<item> -> /<driveletter>/<item>/item>
+// <driveletter>:/<item>/<item> -> /<driveletter>/<item>/item>
+// paths without drive letter are detected and the drive added
+// /<item>/<item> -> /<current driveletter>/<item>/item>
+// relative paths are translated to absolute with drive letter
+// returns null if the string is illegal
+//
+// The string  must be freed
+//
+// we don't handle paths which start with the "\\?\" thing. We never
+// use the 8 bit version of the win32 apis. That said, we get paths in 8 bit
+// characters from the enclave because there is no "wopen" in linux
+//
+// We always convert in the ocall function.
+//
+void normalize_path(char *path, size_t origlen, char slash)
```

Comment:
Sorry the comment is for oe_win_path_to_posix() then it make confusions here. This function have slash as parameter in case it be reused to deal with Windows path.